engines/winnative/payload-schema.yml

```diff
@@ -0,0 +1,57 @@
+id: "http://schemas.taskcluster.net/generic-worker/v1/payload.json#"
+$schema: "http://json-schema.org/draft-04/schema#"
+title: "Payload"
+description: "This schema defines the structure of the `payload` property referred to in a Task Cluster Task definition."
+type: "object"
+required: 
+  - "command"
+  - "maxRunTime"
+properties: 
+  command: 
+    title: "Commands to run"
+    type: "array"
+    minItems: 1
+    items: 
+      type: "array"
+      minItems: 1
+      items: 
+        type: "string"
+    description: "One array per command (each command is an array of arguments). Several arrays for several commands."
+  env: 
+    title: "Environment variable mappings."
+    description: "Example: ```{ \"PATH\": \"C:\\\\Windows\\\\system32;C:\\\\Windows\", \"GOOS\": \"darwin\" }```"
+    type: "object"
+  maxRunTime: 
+    type: "integer"
+    title: "Maximum run time in seconds"
+    description: "Maximum time the task container can run in seconds"
+    multipleOf: 1
+    minimum: 1
+    maximum: 86400
+  artifacts: 
```

Comment:
sorry this is the place to say artifacts aren't part of the engine payload... they are handled by plugins.